components/src/core/ListItemTag/ListItemTag.tsx

```diff
@@ -24,39 +35,41 @@ export type ListItemTagProps = TypographyProps & {
 };
 
 // See https://material-ui.com/guides/right-to-left/#opting-out-of-rtl-transformation
-declare module '@mui/styles/withStyles' {
-    // Augment the BaseCSSProperties so that we can control jss-rtl
-    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-    interface BaseCSSProperties {
-        /**
-         * Used to control if the rule-set should be affected by rtl transformation
-         */
-        flip?: boolean;
-    }
-}
+// declare module '@mui/styles/withStyles' {
+//     // Augment the BaseCSSProperties so that we can control jss-rtl
+//     // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+//     interface BaseCSSProperties {
+//         /**
+//          * Used to control if the rule-set should be affected by rtl transformation
+//          */
+//         flip?: boolean;
+//     }
+// }
```

Comment:
If this is no longer needed can we remove it? How does MUI 5 style engine handle RTL style flipping?